lib/parse.js

```diff
@@ -215,7 +218,13 @@ function createOption(label, value) {
   if (!name) {
     name = label;
   }
-  if (!value) {
+  if (!label) {
+    throw new ParseError({
+      code: 'invalidSelectLabel',
+      message: 'Option label is empty'
+    });
+  }
+  if (value == null) {
```

Comment:
This allows authors to specify an empty string as the value with the object syntax. There is no test for it yet.